test/files/neg/xlint-captured.flags

```diff
@@ -0,0 +1 @@
+-Xfatal-warnings -Xlint:captured
```

Comment:
Should be `// scalac: -Werrror etc` in the test source.

Wait, you know this, what year is this?

Is this that thing where you PR against an old branch and you never know what features are supported?

Reply:
It's the thing where I'm on my phone hotspot and Seth just upgraded sbt so I can't switch to 2.13.x, and I didn't really want to go poking around in git to see what the latest 2.13.x commit I can build on is, and moreover I do know this because I keep telling myself I'll backport it to the scala-partest repo and bribe a kind Lightbender to release a new partest, or I could just not bother.

I get this from you every time, btw. 

Do we even want this in 2.12? @mkeskells might, or @diesalbla.

Reply:
`-Werrror` should be the variant that deletes the source file because it emitted a warning.